pkg/cmd/openshift/openshift.go

```diff
@@ -46,12 +45,6 @@ Note: This is a beta release of OpenShift and may change significantly.  See
 func CommandFor(basename string) *cobra.Command {
 	var cmd *cobra.Command
 
-	// Make case-insensitive and strip executable suffix if present
```

Comment:
Did you intend to remove this section?  looks like an inadvertent mistake when rebasing.